docs/pages/bare/upgrade.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Upgrade your bare project
+description: View file-by-file diffs of all the changes you need to make to your native projects to upgrade them to the next Expo SDK version.
+sidebar_title: Upgrading your bare project
```

Comment:
```suggestion
```

I don't think we need to add a sidebar title that uses Gerund. Lately I've been cleaning this up (WIP) from our docs pages. I'd suggest to remove this.

<img width="260" alt="CleanShot 2023-10-25 at 15 40 58@2x" src="https://github.com/expo/expo/assets/10234615/4545ab27-90d8-44cb-9246-128b6ab65467">